packages/webpack/src/config/base.js

```diff
@@ -207,6 +207,7 @@ export default class WebpackBaseConfig {
             condition: 'some',
             filename: 'LICENSES'
           },
+          sourceMap: this.devtool && /((inline|hidden|nosources)-)?source-?map/.test(this.devtool),
```

Comment:
Can't we normalize and remove # instead?

Reply:
There should also have other variants, hard to tell if we can cover all

Reply:
I'm not sure if we have other variants. But currently, we silently disable sourceMap which is not ideal imo. Maybe we can make a getter that tries to normalize and remove hash and in case it was still invalid, throw a console warn?

Reply:
I also think it would be better to just remove `#` as otherwise this code potentially creates extra maintenance (validating supported values that can change at some point)